wagtailnhsukfrontend/templates/wagtailnhsukfrontend/image.html

```diff
@@ -0,0 +1,37 @@
+{% load wagtailimages_tags %}
+
+{% image value.content_image original as org_image %}
```

Comment:
Don't use the original image. This could be massive!
Select a reasonable default to use for `src=""`. If there was no such thing as the `sizes` attribute, what image size would you select?

Reply:
👍